app/utils/documentation_reader.rb

```diff
@@ -6,7 +6,7 @@
 #  https://github.com/hitobito/hitobito.
 
 class DocumentationReader
-  GITHUB_DEV_DOC_BASE_URL = "https://github.com/hitobito/hitobito/tree/master/doc/developer"
+  GITHUB_DEV_DOC_BASE_URL = "https://github.com/hitobito/hitobito/tree/master"
```

Comment:
Are there other usages of this md link resolver that need to be adjusted? I just found the ones in `json_api.md`.